cirq/vis/heatmap.py

```diff
@@ -63,8 +56,7 @@ class Heatmap:
     def __init__(self, value_map: ValueMap) -> None:
         self.set_value_map(value_map)
         self.annot_map = {  # Default annotation.
-            _get_qubit_row_col(qubit): format(float(value), '.2g')
-            for qubit, value in value_map.items()
+            (*qubit,): format(float(value), '.2g') for qubit, value in value_map.items()
```

Comment:
I think `tuple(qubit)` is more clear. Though a method to explicitly convert to tuple would be better since it could be typed as `Tuple[int, int]` whereas this would give `Tuple[int, ...]` with an unknown number of args.